src/main/java/address/util/OrderedList.java

```diff
@@ -0,0 +1,73 @@
+package address.util;
+
+import address.model.datatypes.person.ReadOnlyViewablePerson;
+import com.sun.javafx.collections.SourceAdapterChange;
+import javafx.collections.FXCollections;
+import javafx.collections.ListChangeListener;
+import javafx.collections.ObservableList;
+import javafx.collections.transformation.TransformationList;
+
+/**
+ *
+ */
+public class OrderedList<T> extends TransformationList<T, T> {
+
+
+    private ObservableList<T> orderedList;
+
+    /**
+     * Creates a new Transformation list wrapped around the source list.
+     *
+     * @param source the wrapped list
+     */
+    public OrderedList(ObservableList<T> source) {
+        super(source);
+        orderedList = FXCollections.observableArrayList();
```

Comment:
copy the empty list to be safe.
orderedList = FXCollections.observableArrayList(source);
Since it may not be empty in other scenario if we happen to use this OrderedList on other application